src/index.html

```diff
@@ -31,8 +30,7 @@ <h3>Migrate an existing <br />Zigbee network</h3>
       <div></div>
     </div>
     <div>
-      <h3>Thread and Matter</h3>
-      <p><a href="https://www.home-assistant.io/skyconnect">Home Assistant SkyConnect</a> comes with experimental support to power Thread (used for some Matter devices) and
-        Zigbee networks at the same time. Before <a href="/procedures/enable-multiprotocol/">enabling multiprotocol support</a>, check the section about <a href="/about-multiprotocol/">multiprotocol issues</a>.</p>
+      <h3>Thread</h3>
+      <p><a href="https://www.home-assistant.io/skyconnect">Home Assistant SkyConnect</a> can also be used to turn your Home Assistant installation into a Thread border router. Currently, enabling Thread involves manual steps. The integration of the Home Assistant Thread border router with Matter is work-in-progress. To learn more about Thread border routers, refer to the <a href="https://www.home-assistant.io/integrations/thread/#list-of-thread-border-router-devices">List of Thread border routers</a>. To learn how to use SkyConnect to turn your Home Assistant into a Thread Border router, refer to <a href="https://www.home-assistant.io/integrations/thread/#turning-home-assistant-into-a-thread-border-router">Turning Home Assistant into a Thread border router</a>.</p>
```

Comment:
Actually, on second thought, it is a bit weird that we link back to the marketing page here once again. Let's drop this.

```suggestion
      <p>Home Assistant SkyConnect can also be used to turn your Home Assistant installation into a Thread border router. Currently, enabling Thread involves manual steps. The integration of the Home Assistant Thread border router with Matter is work-in-progress. To learn more about Thread border routers, refer to the <a href="https://www.home-assistant.io/integrations/thread/#list-of-thread-border-router-devices">List of Thread border routers</a>. To learn how to use SkyConnect to turn your Home Assistant into a Thread Border router, refer to <a href="https://www.home-assistant.io/integrations/thread/#turning-home-assistant-into-a-thread-border-router">Turning Home Assistant into a Thread border router</a>.</p>
```